src/hotspot/share/opto/loopTransform.cpp

```diff
@@ -968,6 +970,11 @@ bool IdealLoopTree::policy_unroll(PhaseIdealLoop *phase) {
       case Op_ModL: body_size += 30; break;
       case Op_DivL: body_size += 30; break;
       case Op_MulL: body_size += 10; break;
+      case Op_PopCountVI:
+      case Op_PopCountVL: {
+        const TypeVect * vt = n->bottom_type()->is_vect();
+        body_size += Matcher::vector_op_cost(n->Opcode(), vt->element_basic_type(), vt->length());
```

Comment:
For other platforms, I think `vector_op_cost` should be better to return 0 at this moment.

Reply:
I thought the total cost has been counted by `uint body_size = _body.size()`. So that for other platforms, the total cost now is one more if counted PopCountVI/VL.